apps/govquests-frontend/src/domains/questing/components/QuestDetails.tsx

```diff
@@ -12,41 +15,65 @@ interface QuestDetailsProps {
 const QuestDetails: React.FC<QuestDetailsProps> = ({ quest }) => {
   if (!quest) return null;
 
+  const { isConnected } = useAccount();
+  const { isSignedIn, signIn } = useSIWE();
+
   const status = quest.userQuests?.[0]?.status || "unstarted";
 
   return (
-    <main className="flex items-center justify-center h-full">
-      <div className="flex flex-col max-w-[85%]">
-        <div className="flex flex-1 items-center p-5 md:p-8 bg-black/60 text-optimismForeground rounded-lg mb-3">
-          <MapIcon width={50} height={50} />
-          <h1 className="text-3xl flex-1 ml-5">{quest.displayData.title}</h1>
-          <div className="flex items-center">
-            {quest.rewards.map((reward) => (
-              <RewardIndicator key={reward.type} reward={reward} />
-            ))}
-          </div>
-        </div>
-        <div className="flex flex-col flex-1 justify-center bg-primary p-5 md:p-8 rounded-lg">
-          <div className="flex items-center">
-            <div className="w-24 h-24 bg-optimism" />
-            <div className="ml-7">
-              <h2 className="text-2xl font-medium mb-2">About this quest</h2>
-              <p className="text-lg">{quest.displayData.intro}</p>
+    <main className="flex justify-center h-full">
+      <div className="flex flex-col w-[70%] mt-4">
+        <Link
+          href="#"
+          onClick={(e) => {
+            e.preventDefault();
+            window.history.back();
+          }}
```

Comment:
🤔 what if this is the first page they've opened? IMO this should go back to the quest list